### PR TITLE
updates to easyconfigs for GCC 10.2.0 and compatability to build GRASS and QGIS

### PR DESCRIPTION
**Pull request by jedokaplan**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GTK+/GTK+-3.24.23-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -53,11 +54,8 @@ components = [
     ('adwaita-icon-theme', '3.38.0', {
         'preconfigopts': 'autoreconf -f -i && ',
         'source_urls': [FTPGNOME_SOURCE],
-        'patches': ['adwaita-icon-theme-3.34.3_disable-svg-conversion.patch'],
         'checksums': [
             '6683a1aaf2430ccd9ea638dd4bfe1002bc92b412050c3dba20e480f979faaf97',
-            # adwaita-icon-theme-3.34.3_disable-svg-conversion.patch
-            'f4b86855d50759ecfc1e8f6550ec0f3a7a4ea2c80b9f5fc1685fe8967d1c5342',
         ],
```
Comment (Micket) [inline easybuild/easyconfigs/g/GTK+/GTK+-3.24.23-GCCcore-10.2.0.eb:59]: Please don't remove these.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.9.1-GCC-10.2.0-Python-3.8.6.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'GEOS'
```
Comment (Micket) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.9.1-GCC-10.2.0-Python-3.8.6.eb:3]: Already exists

File: easybuild/easyconfigs/l/libcroco/libcroco-0.6.13-GCCcore-10.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libcroco'
```
Comment (Micket) [inline easybuild/easyconfigs/l/libcroco/libcroco-0.6.13-GCCcore-10.2.0.eb:3]: Hm, well, this has a version at GCC level already, though, I suppose moving it to core is reasonable (not sure why it wasn't from the start)

File: easybuild/easyconfigs/l/librsvg/librsvg-2.50.7-GCCcore-10.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'librsvg'
```
Comment (Micket) [inline easybuild/easyconfigs/l/librsvg/librsvg-2.50.7-GCCcore-10.2.0.eb:3]: You are adding both a GCC, and a GCCcore version

File: easybuild/easyconfigs/l/librsvg/librsvg-2.50.7-foss-2020b.eb [hunk 5]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'librsvg'
```
Comment (Micket) [inline easybuild/easyconfigs/l/librsvg/librsvg-2.50.7-foss-2020b.eb:3]: and a foss version as well

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-13.3-GCCcore-10.2.0.eb [hunk 6]
```diff
@@ -0,0 +1,50 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'ConfigureMake'
+
+name = 'PostgreSQL'
+version = '13.3'
```
Comment (Micket) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-13.3-GCCcore-10.2.0.eb:7]: version 13.2 already exists, I doubt this minor version bump is necessary

File: easybuild/easyconfigs/p/PyQt5/PyQt5-5.15.4-GCCcore-10.2.0-Python-3.8.6.eb [hunk 7]
```diff
@@ -0,0 +1,108 @@
+easyblock = 'Bundle'
+
+name = 'PyQt5'
+version = '5.15.4'
```
Comment (Micket) [inline easybuild/easyconfigs/p/PyQt5/PyQt5-5.15.4-GCCcore-10.2.0-Python-3.8.6.eb:4]: Version 5.15.1 already exists, i doubt this minor version bump is necessary

File: easybuild/easyconfigs/q/Qt5Webkit/Qt5Webkit-5.212.0-alpha4-GCCcore-10.2.0.eb [hunk 8]
```diff
@@ -53,4 +53,4 @@ sanity_check_paths = {
     'dirs': ['include/QtWebKit', 'include/QtWebKitWidgets'],
 }
 
-moduleclass = 'devel'
```
Comment (Micket) [inline easybuild/easyconfigs/q/Qt5Webkit/Qt5Webkit-5.212.0-alpha4-GCCcore-10.2.0.eb:56]: Please exclude this change

**Review by Micket — CHANGES_REQUESTED** [review]
In the newer toolchain versions, we decided to drop the python suffix, considering Python 3 to be the default. 

Another general comment, is that a PR like this much to large to be pratical to review. Especially with monster builds like Qt5 inside here, making them very impractical to just upload a test report.

**Comment by boegel** [comment]
closing this since `GCC/10.2.0` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**